### PR TITLE
fix(components): cast return value of gcp/automl/create_dataset_for_tables. Fixes #5053

### LINKED ISSUE #5053: TypeErro occurs in gcp/automl/create_dataset_for_tables component
### What steps did you take:
[A clear and concise description of what the bug is.]

[gcp/automl/create_dataset_for_tables component](https://github.com/kubeflow/pipelines/tree/master/components/gcp/automl/create_dataset_for_tables)'s `create_time` output is declared as a string:

https://github.com/kubeflow/pipelines/blob/ecb14f40bb819c0678589b6458892ece5369fa71/components/gcp/automl/create_dataset_for_tables/component.yaml#L15

however, `google.protobuf.timestamp_pb2.Timestamp` is returned in actual fact:

https://github.com/kubeflow/pipelines/blob/ecb14f40bb819c0678589b6458892ece5369fa71/components/gcp/automl/create_dataset_for_tables/component.py#L54

FYI: The `dataset` object is an instance of `google.cloud.automl_v1beta1.types.Dataset` class and its [document](https://googleapis.dev/python/automl/0.4.0/gapic/v1beta1/types.html#google.cloud.automl_v1beta1.types.Dataset.create_time) says:

> **create_time**
> Output only. Timestamp when this dataset was created.

### What happened:

`TypeError` occurs

image

### What did you expect to happen:

Work.

### Environment:

How did you deploy Kubeflow Pipelines (KFP)? AI Platform Pipelines

KFP version: 1.0.4 

KFP SDK version: 1.3.0 

### Anything else you would like to add:
[Miscellaneous information that will assist in solving the issue.]

/kind bug

### PR DESCRIPTION
**Pull request by yuku**
**Description of your changes:**
Cast return value of gcp/automl/create_dataset_for_tables. Fixes #5053 

**Quality assurance**
I confirmed that the patched component.yaml fixes the issue.

_Before_
image

_After: Loading the patched component.yaml. (same inputs)_
image

</details>

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [x] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

File: components/gcp/automl/create_dataset_for_tables/component.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def automl_create_dataset_for_tables(
         region=gcp_region,
         dataset_id=dataset_id,
     )
-    return (dataset.name, dataset.create_time, dataset_id, dataset_url)
+    return (dataset.name, str(dataset.create_time), dataset_id, dataset_url)
```
Comment (numerology) [inline components/gcp/automl/create_dataset_for_tables/component.py:54]: Do you mind dropping a comment about why this casting is needed? Thanks!

Comment (yuku) [reply components/gcp/automl/create_dataset_for_tables/component.py:54]: While writing a comment, the @google-oss-robot merged the PR 😂 
Should I open another PR?

Comment (numerology) [reply components/gcp/automl/create_dataset_for_tables/component.py:54]: My bad🤣 I forgot the auto-merge.

Feel free to do that once you have a chance. Thanks!

**Pull request merged**